### DISCUSSION TITLE
Cypress.io - report and run status set up

### DISCUSSION THREAD

**Discussion by faprile** [cypress-io/cypress#14777] [CI setup] [answered]
Hello everybody and thanks to all in advance, I'm a newbie with Cypress (I practised the automation in my previous experiences but with other tool and languages). I started yesterday to set up it and to write my first UI tests. I have no problem with the tests writing but I've some questions about the report set up and the run status set up.

First, during the report set up, I wrote this in the cypress.io file:

`{ "reporter": "spec" }`

but I don't know how this work. How can I use the "spec" report? What do I need to see the report?

Second, I tried to configure my project with the Cypress Analytics (that can I use it as a report?). In the cypress.json file, I wrote my project ID, but I've got a problem when I try to execute the command:` npm cypress run --record --key xxxxxxxxxx `because I receive this message:

```
Usage: npm <command>

where <command> is one of:
access, adduser, audit, bin, bugs, c, cache, ci, cit,
clean-install, clean-install-test, completion, config,
create, ddp, dedupe, deprecate, dist-tag, docs, doctor,
edit, explore, fund, get, help, help-search, hook, i, init,
install, install-ci-test, install-test, it, link, list, ln,
login, logout, ls, org, outdated, owner, pack, ping, prefix,
profile, prune, publish, rb, rebuild, repo, restart, root,
run, run-script, s, se, search, set, shrinkwrap, star,
stars, start, stop, t, team, test, token, tst, un,
uninstall, unpublish, unstar, up, update, v, version, view,
whoami

npm <command> -h quick help on <command>
npm -l display full usage info
npm help <term> search for help on <term>
npm help npm involved overview

Specify configs in the ini-formatted file:
```

**Comment by bahmutov** [comment, marked as answer]
Instead of `npm cypress run --record --key xxxxxxxxxx` use `npx cypress run --record --key xxxxxxxxxx` command. See https://on.cypress.io/command-line#How-to-run-commands

**Reply by faprile** [reply]
Thanks! Now it works.
